datafusion/expr/src/expr.rs

```diff
@@ -693,9 +689,6 @@ impl AggregateFunction {
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Hash)]
 /// Defines which implementation of an aggregate function DataFusion should call.
 pub enum WindowFunctionDefinition {
-    /// A built in aggregate function that leverages an aggregate function
-    /// A a built-in window function
-    BuiltInWindowFunction(BuiltInWindowFunction),
```

Comment:
Let us keep this PR limited to converting `nth_value`. We can handle cleanup of `BuiltInWindowFunction` in a follow-up PR.